From 37fab3bd51a88950121d1f49dc654e769d919075 Mon Sep 17 00:00:00 2001
From: Iurii Konovalenko <iurii.konovalenko@globallogic.com>
Date: Thu, 9 Jun 2016 14:20:58 +0300
Subject: [PATCH 1/3] clk:shmobile: Hide clocks not used in DomD

On Renesas Salvaotor board with Xen SCIF2 is provided to hypervisor, so
kernel should not manage SCIF2 clock.
Also, some hardware is provided to other domains, so kernel should mot
manage their clocks.

Signed-off-by: Iurii Konovalenko <iurii.konovalenko@globallogic.com>
---
 drivers/clk/shmobile/r8a7795-cpg-mssr.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/clk/shmobile/r8a7795-cpg-mssr.c b/drivers/clk/shmobile/r8a7795-cpg-mssr.c
index 8f180fd..79ceb0e 100644
--- a/drivers/clk/shmobile/r8a7795-cpg-mssr.c
+++ b/drivers/clk/shmobile/r8a7795-cpg-mssr.c
@@ -90,7 +90,7 @@ static const struct cpg_core_clk r8a7795_core_clks[] __initconst = {
 	DEF_FIXED(".s1",        CLK_S1,            CLK_PLL1_DIV2,  3, 1),
 	DEF_FIXED(".s2",        CLK_S2,            CLK_PLL1_DIV2,  4, 1),
 	DEF_FIXED(".s3",        CLK_S3,            CLK_PLL1_DIV2,  6, 1),
-	DEF_FIXED(".sdsrc",     CLK_SDSRC,         CLK_PLL1_DIV2,  2, 1),
+	/*DEF_FIXED(".sdsrc",     CLK_SDSRC,         CLK_PLL1_DIV2,  2, 1),*/
 
 	/* Core Clock Outputs */
 	DEF_FIXED("zg",         R8A7795_CLK_ZG,    CLK_PLL4,       2, 1),
@@ -113,10 +113,10 @@ static const struct cpg_core_clk r8a7795_core_clks[] __initconst = {
 	DEF_FIXED("cp",         R8A7795_CLK_CP,    CLK_EXTAL,      2, 1),
 
 	DEF_BASE("r",           R8A7795_CLK_R,   CLK_TYPE_GEN3_R,   CLK_EXTAL),
-	DEF_BASE("sd0",         R8A7795_CLK_SD0, CLK_TYPE_GEN3_SD0, CLK_SDSRC),
+	/*DEF_BASE("sd0",         R8A7795_CLK_SD0, CLK_TYPE_GEN3_SD0, CLK_SDSRC),
 	DEF_BASE("sd1",         R8A7795_CLK_SD1, CLK_TYPE_GEN3_SD1, CLK_SDSRC),
 	DEF_BASE("sd2",         R8A7795_CLK_SD2, CLK_TYPE_GEN3_SD2, CLK_SDSRC),
-	DEF_BASE("sd3",         R8A7795_CLK_SD3, CLK_TYPE_GEN3_SD3, CLK_SDSRC),
+	DEF_BASE("sd3",         R8A7795_CLK_SD3, CLK_TYPE_GEN3_SD3, CLK_SDSRC),*/
 	DEF_BASE("z",           R8A7795_CLK_Z,   CLK_TYPE_GEN3_Z,   CLK_PLL0),
 	DEF_BASE("z2",          R8A7795_CLK_Z2,  CLK_TYPE_GEN3_Z2,  CLK_PLL2),
 
@@ -147,11 +147,11 @@ static const struct mssr_mod_clk r8a7795_mod_clks[] __initconst = {
 	DEF_MOD("sys-dmac2",		 217,	R8A7795_CLK_S3D1),
 	DEF_MOD("sys-dmac1",		 218,	R8A7795_CLK_S3D1),
 	DEF_MOD("sys-dmac0",		 219,	R8A7795_CLK_S3D1),
-	DEF_MOD("scif2",		 310,	R8A7795_CLK_S3D4),
+	/*DEF_MOD("scif2",		 310,	R8A7795_CLK_S3D4),
 	DEF_MOD("sdhi3",		 311,	R8A7795_CLK_SD3),
 	DEF_MOD("sdhi2",		 312,	R8A7795_CLK_SD2),
 	DEF_MOD("sdhi1",		 313,	R8A7795_CLK_SD1),
-	DEF_MOD("sdhi0",		 314,	R8A7795_CLK_SD0),
+	DEF_MOD("sdhi0",		 314,	R8A7795_CLK_SD0),*/
 	DEF_MOD("pcie1",		 318,	R8A7795_CLK_S3D1),
 	DEF_MOD("pcie0",		 319,	R8A7795_CLK_S3D1),
 	DEF_MOD("usb3-if1",		 327,	R8A7795_CLK_S3D1),
-- 
1.9.1

